test/test_irods_http_api.py

```diff
@@ -727,32 +881,36 @@ def test_registering_a_new_data_object(self):
             self.assertEqual(result['rows'][0][3], resource)
 
         finally:
-            # Remove the data object by unregistering it.
+            # Unregister the data object.
             r = requests.post(self.url_endpoint, headers=rodsadmin_headers, data={
                 'op': 'remove',
                 'lpath': data_object,
-                'unregister': 1
+                'catalog-only': 1
             })
-            #print(r.content) # Debug
+            logging.debug(r.content)
 
     def test_registering_an_additional_replica_for_an_existing_data_object(self):
         rodsadmin_headers = {'Authorization': 'Bearer ' + self.rodsadmin_bearer_token}
 
         # The name of the resource we'll use to register an additional replica under.
         other_resource = 'test_registration_resc'
 
+        # The basename and logical path of the data object.
+        filename = 'test_registering_replica.txt'
+        data_object = f'/{self.zone_name}/home/{self.rodsadmin_username}/{filename}'
+
+        # The name of the physical file to create and register as an additional replica.
+        physical_path = f'/tmp/{filename}'
```

Comment:
Consider using `tempfile.NamedTemporaryFile`: https://docs.python.org/3.8/library/tempfile.html?highlight=tempfile#tempfile.NamedTemporaryFile Don't have to, just putting it out there in case it might present a better alternative.

Reply:
I'll handle this in a separate PR.

Created #204 for this.